helm_deploy/values-preprod.yaml

```diff
@@ -15,3 +15,4 @@ generic-service:
     WORKLOAD_SERVICE_URL: "https://hmpps-workload-preprod.hmpps.service.justice.gov.uk"
     PROBATION_ESTATE_SERVICE_URL: "https://hmpps-probation-estate-api-preprod.hmpps.service.justice.gov.uk"
     USER_PREFERENCE_SERVICE_URL: "https://hmpps-user-preferences-preprod.hmpps.service.justice.gov.uk"
+    OASYS_URL: "https://bridge-oasys.az.justice.gov.uk"
```

Comment:
this looks like it is the same as the production URL - is there no preprod instance?

Reply:
I couldn't find any reference to a preprod url so set it to the prod url for now